resources/views/dashboard.blade.php

```diff
@@ -0,0 +1,200 @@
+@extends('layouts.main')
+
+@section('title', 'Dashboard')
+
+@section('css')
+    @vite(['resources/css/dashboard.css', 'resources/js/gradePass.js'])
```

Comment:
In real world situations, page specific CSS is avoided as much as possible in order to maintain and keep a coherent user experience and avoid code duplication. It is usually achieved by rethinking the way components and styling is structured. Take a look at how CSS libraries like Bulma or Bootstrap solve this if you're interested.